3-platform/shopping-cart/infra/src/main/java/com/_6core/platform/shopping/cart/infra/dto/CreateOrderRequest.java

```diff
@@ -0,0 +1,5 @@
+package com._6core.platform.shopping.cart.infra.dto;
+
+import java.util.List;
+
+public record CreateOrderRequest(List<String> product_ids) {}
```

Comment:
pls dont forget about Java convention 
```suggestion
public record CreateOrderRequest(List<String> productIds) {}
```